timelock-server/src/suiteTest/java/com/palantir/atlasdb/timelock/MultiNodePaxosTimeLockServerIntegrationTest.java

```diff
@@ -144,55 +145,55 @@ public void nonLeadersReturn503() {
 
     @Test
     public void leaderRespondsToRequests() {
-        CLUSTER.currentLeader().getFreshTimestamp();
+        cluster.currentLeader().getFreshTimestamp();
 
-        LockToken token = CLUSTER.currentLeader().lock(LockRequest.of(LOCKS, DEFAULT_LOCK_TIMEOUT_MS)).getToken();
-        CLUSTER.unlock(token);
+        LockToken token = cluster.currentLeader().lock(LockRequest.of(LOCKS, DEFAULT_LOCK_TIMEOUT_MS)).getToken();
+        cluster.unlock(token);
     }
 
     @Test
     public void newLeaderTakesOverIfCurrentLeaderDies() {
-        CLUSTER.currentLeader().kill();
+        cluster.currentLeader().kill();
 
-        CLUSTER.getFreshTimestamp();
+        cluster.getFreshTimestamp();
     }
 
     @Test
     public void leaderLosesLeadershipIfQuorumIsNotAlive() {
-        TestableTimelockServer leader = CLUSTER.currentLeader();
-        CLUSTER.nonLeaders().forEach(TestableTimelockServer::kill);
+        TestableTimelockServer leader = cluster.currentLeader();
+        cluster.nonLeaders().forEach(TestableTimelockServer::kill);
 
         assertThatThrownBy(leader::getFreshTimestamp)
                 .satisfies(ExceptionMatchers::isRetryableExceptionWhereLeaderCannotBeFound);
     }
 
     @Test
     public void someoneBecomesLeaderAgainAfterQuorumIsRestored() {
-        CLUSTER.nonLeaders().forEach(TestableTimelockServer::kill);
-        CLUSTER.nonLeaders().forEach(TestableTimelockServer::start);
+        cluster.nonLeaders().forEach(TestableTimelockServer::kill);
+        cluster.nonLeaders().forEach(TestableTimelockServer::start);
 
-        CLUSTER.getFreshTimestamp();
+        cluster.getFreshTimestamp();
     }
 
     @Test
     public void canPerformRollingRestart() {
         bringAllNodesOnline();
-        for (TestableTimelockServer server : CLUSTER.servers()) {
+        for (TestableTimelockServer server : cluster.servers()) {
             server.kill();
-            waitForClusterToStabilize();
-            CLUSTER.getFreshTimestamp();
+            cluster.waitUntilAllServersOnlineAndReadyToServeClients(ADDITIONAL_CLIENTS);
```

Comment:
I think this should be `cluster.waitUntilLeaderIsElected()`. This one actually starts the dead servers again (before we get the fresh timestamp), which is probably not what the test wants to do